pkgs/applications/video/gnome-mpv/default.nix

```diff
@@ -5,21 +5,24 @@
 
 stdenv.mkDerivation rec {
   name = "gnome-mpv-${version}";
-  version = "0.13";
+  version = "0.16";
 
   src = fetchFromGitHub {
     owner = "gnome-mpv";
     repo = "gnome-mpv";
-    rev = "0d73b33d60050fd32bf8fae77d831548970a0b69"; # upstream forgot to update appdata
-    # rev = "v${version}";
-    sha256 = "1cjhw3kz163iwj2japhnv354i1lr112xyyfkxw82cwy2554cfim4";
+    rev = "v${version}";
+    sha256 = "1fj5mr1dwd07jpnigk7z85xdm6yaf7spbvf60aj3mz12m05b1b2w";
   };
 
   nativeBuildInputs = [ meson ninja python3 appstream-glib gettext pkgconfig desktop-file-utils wrapGAppsHook ];
   buildInputs = [ epoxy glib gtk3 mpv ];
 
   enableParallelBuilding = true;
```

Comment:
This is not necessary with Meson.

Reply:
What do you mean? `fetchFromGithub` will need to know what rev to fetch.

Reply:
Sorry, GitHub does not make it clear that I meant `enableParallelBuilding = true;`. Meson enables it by default so we can clean it up.

Reply:
Ah, makes sense. I'll clean that up.